localization/english/wc_game_concepts_l_english.yml

```diff
@@ -63,4 +63,24 @@
 
  game_concept_eldritch_government:0 "$eldritch_government_adjective$ Government"
- game_concept_eldritch_government_desc:0 "A [dark_government|E] available to [GetTrait('being_void').GetName( GetNullCharacter )] with $doctrine_shadow_magic_name$ accepted.\nHaving high [dark_authority|E], they get all [skills|E] per [stress_level|E] and increase [county_development_growth|E]."
\ No newline at end of file
+ game_concept_eldritch_government_desc:0 "A [dark_government|E] available to [GetTrait('being_void').GetName( GetNullCharacter )] with $doctrine_shadow_magic_name$ accepted.\nHaving high [dark_authority|E], they get all [skills|E] per [stress_level|E] and increase [county_development_growth|E]."
+ 
+ game_concept_scourge_jailor:0 "Scourge Jailor"
+ game_concept_scourge_jailors:0 "Scourge Jailors"
+ game_concept_scourge_jailor_and_damned_common_desc:0 "As long as the [scourge_jailors|E] alive, the [scourge|E] is guaranteed to stay a [legions_puppet|E]. Any public hostile action of the [scourge_damned|E] against a [scourge_jailor|E] like [imprisoning|E] or open [murder|E] will result in [execution|E]."
+ game_concept_scourge_jailor_desc:0 "A [relation|E] indicating a [GetTrait('creature_nathrezim').GetName( GetNullCharacter )] sent to oversee the [scourge_damned|E].\n\n$game_concept_scourge_jailor_and_damned_common_desc$"
+ 
+ game_concept_legions_puppet:0 "Legion's Puppet"
+ game_concept_legions_puppet_desc:0 "A state of the [scourge|E] where they obey the Legion's commands. While it's a case, $deathbringer_adj$ and $doombringer_adj$ [faiths|E] consider each other [faith_astray|E].\n\nThe [GetTrait('lich_king').GetName( GetNullCharacter )] has 15 years to summon the Legion. If the [GetTrait('lich_king').GetName( GetNullCharacter )] fails, [scourge_jailors|E] end the [scourge|E]. If all the [scourge_jailors|E] are dead or imprisoned or the Legion is summoned, the [scourge|E] is free and can survive on its own."
+ 
+ game_concept_scourge:0 "Scourge"
+ game_concept_scourge_possessive:0 "Scourge's"
+ game_concept_scourge_desc:0 "One of the main threats on Azeroth. The $game_concept_scourge$ has a [necro_government|E] and is led by the [GetTrait('lich_king').GetName( GetNullCharacter )], a powerful necromancer who controls their [vassals|E] with an iron grip.\n\nIt enables [great_invasion_casus_belli|E] but instead of taking lands you take vassals and turn them [GetTrait('being_undead').GetName( GetNullCharacter )].\n\nThe $game_concept_scourge$ starts as a [legions_puppet|E] thus the first 15 years are dedicated to bringing the Legion into Azeroth."
+ 
+ game_concept_great_invasion_casus_belli:0 "Great Invasion Casus Belli"
+ game_concept_great_invasion_casus_belli_desc:0 "A powerful [casus_belli|E] avaliable to some crises.\n\nIt can target an entire [kingdom|E] [title|E] at once, and allows for rapid expansion across large swathes of land."
+ 
+ game_concept_obey_legion:0 "Obey the Legion"
+ 
+ game_concept_scourge_damned:0 "Scourge Damned"
```

Comment:
Shouldn't this be Scourge of the Damned?

Reply:
I don't think so.
It's a relation (like a friend/rival), this one is a relation shown when you look as LK from nathrezim's POV.

Reply:
Scourge is Damned then? The way it is, I can't make any sense of it how to translate it...

Reply:
I'm afraid then the relationship tooltip will be read as "Your Liege and Scourge is Damned"

Reply:
I changed it in e3f9159ae286f5daf12bf7b03d6a3216cc27152e.